tools/cli/pkg/command/runtime.go

```diff
@@ -116,21 +117,29 @@ func (cmd *RuntimeCommand) Run() error {
 		return errors.Wrap(err, "while listing runtimes")
 	}
 	var eventList []events.EventDTO
+	var eventsSkipped bool
 	if cmd.params.Events && rp.Count > 0 {
-		ev := events.NewClient(GlobalOpts.KEBAPIURL(), httpClient)
-		var instanceIds []string
-		for _, i := range rp.Data {
-			instanceIds = append(instanceIds, i.InstanceID)
-		}
-		eventList, err = ev.ListEvents(instanceIds)
-		if err != nil {
-			return errors.Wrap(err, "while listing events")
+		if rp.Count > 100 {
+			eventsSkipped = true
+		} else {
+			ev := events.NewClient(GlobalOpts.KEBAPIURL(), httpClient)
+			var instanceIds []string
+			for _, i := range rp.Data {
+				instanceIds = append(instanceIds, i.InstanceID)
+			}
+			eventList, err = ev.ListEvents(instanceIds)
+			if err != nil {
+				return errors.Wrap(err, "while listing events")
+			}
 		}
 	}
 	err = cmd.printRuntimes(rp, eventList)
 	if err != nil {
 		return errors.Wrap(err, "while printing runtimes")
 	}
+	if eventsSkipped {
+		fmt.Fprintln(os.Stderr, "\nPlease narrow down the instance list by additional filters. fetching events limitted to 100 instances, received", rp.Count)
```

Comment:
What do you think about printing this info message where `eventsSkipped` is set to true?

Reply:
I had that initially but my problem was it could get buried all the way on the top by thousands of lines of output
```
$ kcp rt --events
Please narrow down the instance list by additional filters. fetching events limitted to 100 instances, received 1895
.....
.....
```
while now it shows on the bottom and is not that easy to miss the information
```
$ kcp rt --events
.....
.....

Please narrow down the instance list by additional filters. fetching events limitted to 100 instances, received 1895
```